DrinkManagerWeb/Views/Drinks/SearchByAlcoholContent.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿@using BLL
+@model DrinkManagerWeb.Models.ViewModels.DrinksViewModel
+
+@{
+    ViewBag.Title = "Drinks";
+    Layout = "_Layout";
+}
+<div class="row px-4">
+    <div class="col-12">
+        <header class="d-flex align-items-center">
+            <h1>Search by Alcohol Content</h1>
+        </header>
+        @*Checkboxes returns user choice of alcohol content to controller. 
+            They are automatically checked when user comes to this view from another view*@
+        <div>
+            <form asp-route-pageNumber="1" method="post">
+                <div class="form-check">
+                    @Html.CheckBoxFor(item => item.Alcoholics, new { @class = "form-check-input", @id = "alcoholicsCheck" })
+                    <label class="form-check-label" for="alcoholicsCheck">Alcoholic Drinks</label>
```

Comment:
I think this: for="alcoholicsCheck" is redundant. There is info that "Cannot resolve id 'alcoholicsCheck'". I removed it here and in two other places down below and program works. But check it by yourself too.

Reply:
Still working, done!